libs/ui-components/src/components/Catalog/AddCatalogItemWizard/steps/VersionStep.tsx

```
@@ -0,0 +1,245 @@
+      />
+    </FormGroup>
+  );
+};
+
+const VersionStep = ({ isReadOnly, isEdit }: { isReadOnly?: boolean; isEdit: boolean }) => {
+  const { t } = useTranslation();
+  const { values } = useFormikContext<AddCatalogItemFormValues>();
+
+  const availableChannels = React.useMemo(() => {
+    const channelSet = new Set<string>();
+    values.versions.forEach((v) => v.channels.forEach((c) => channelSet.add(c)));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)